src/docs/markdown/caddyfile/common-uses.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Common Caddyfile Uses
+---
+
+# Common Caddyfile Uses
+
+This page presents some complete, minimal Caddyfile configurations for common use cases, which you might find helpful when building your own configs.
+
+These are not drop-in solutions; you will have to customize your domain name, ports/sockets, directory paths, etc. They are intended to illustrate only the basic requirements for the most common use cases.
+
+#### Menu
+
+- [Static file server](#static-file-server)
+- [Reverse proxy](#reverse-proxy)
+- [PHP](#php)
+- [Redirect `www.` subdomain](#redirect-www-subdomain)
+- [Trailing slashes](#trailing-slashes)
+
+
+## Static file server
+
+```
+example.com
+
+root * /var/www  # optional; default root is current directory
```

Comment:
I'd leave off the comment here.  Most people run webservers as daemons, not from the command line, so for the "common uses" theme this comment is noise.